include-build/roborazzi-gradle-plugin/src/main/java/io/github/takahirom/roborazzi/WebAssets.kt

```diff
@@ -0,0 +1,78 @@
+package io.github.takahirom.roborazzi
+
+import org.webjars.WebJarVersionLocator
+import java.io.File
+
+class WebAssets private constructor(private val webJarVersionLocator: WebJarVersionLocator) {
```

Comment:
👍

Reply:
Thanks. The CSS from the webjar packing seems a little old. The styling is a little off from `main` Looking to see if there is a recently one published with webjar.

Reply:
I noticed it, but it's a subtle change. So I think it is ok

Reply:
Yeah, however not all the tabs shows. That I think needs to be fixed.

Reply:
Resolved it with [9864d9d](https://github.com/takahirom/roborazzi/pull/404/commits/9864d9d2754de16e8fd7d7ae10439c62088048d3). All the tabs shows now though we lost the fixed sized tabs. I believe it should be okay?